.github/CODEOWNERS

```diff
@@ -19,157 +19,183 @@
 # This should make it easy to add new rules without breaking existing ones.
 
 ######## API landing page ###########
-/api/** @dotnet/docs
+/api/ @dotnet/docs
 
 ############ WHATS NEW ###############
-/docs/whats-new/** @billwagner
+/docs/whats-new/ @billwagner
 
 ############  GUIDES  ################
 # .NET Core
-/docs/core/** @dotnet/docs
+/docs/core/ @dotnet/docs
```

Comment:
I think this is good. It just includes the root level items and assigns that to the team.